lib/taskana-core/src/test/java/acceptance/task/CreateTaskAccTest.java

```diff
@@ -98,17 +98,23 @@ public void testCreateTaskWithPlanned()
 
         TaskService taskService = taskanaEngine.getTaskService();
         Task newTask = taskService.newTask("USER_1_1", "DOMAIN_A");
+        Instant instantPlanned = Instant.now().plus(2, ChronoUnit.HOURS);
         newTask.setClassificationKey("T2100");
         newTask.setPrimaryObjRef(createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", "1234567"));
         newTask.setOwner("user_1_1");
-        newTask.setPlanned(Instant.now().plus(2, ChronoUnit.HOURS));
+        newTask.setPlanned(instantPlanned);
         Task createdTask = taskService.createTask(newTask);
 
         assertNotNull(createdTask);
         assertNotNull(createdTask.getCreated());
         assertNotNull(createdTask.getPlanned());
-        assertEquals(createdTask.getCreated().plus(2, ChronoUnit.HOURS).truncatedTo(ChronoUnit.SECONDS),
-            createdTask.getPlanned().truncatedTo(ChronoUnit.SECONDS));
+        assertEquals(instantPlanned, createdTask.getPlanned());
+        assertTrue(createdTask.getCreated().isBefore(createdTask.getPlanned()));
+
+        //verify that planned takes place 2 hours after creation (+- 5 seconds)
+        long difference = Duration.between(createdTask.getCreated(), createdTask.getPlanned()).abs().getSeconds();
+        //add some tolerance to ignore that "created" depends on execution speed
+        assertTrue(Math.abs(difference - 2 * 60 * 60) < 5);
```

Comment:
what are you doing here exactly? I expected difference to be in seconds already..

Reply:
for better readability: 
subtract 2 hours using https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#minusHours-long-

and then only check if < 5.

what do u think?

Reply:
done